bin/pkgCheck/BUILD

```diff
@@ -0,0 +1,11 @@
+py_binary(
```

Comment:
Maybe calling this CovCheck would be more descriptive?

Or are you thinking that this tool can also eventually grow up and be used to test perf baselines for packages too?